### DISCUSSION TITLE
How do I get accentuation in Linux version?

### DISCUSSION THREAD

**Discussion by jeapf** [freeplane/freeplane#1105] [Help]
Hi there. I am using Freeplane 1.10.5 at my Kubuntu Notebook and I am brazilian so my native language is PT-BR. As so I use accentuation but I can't find a way to use it at Kubuntu Freeplane as easy as I use at Windows version.
How can I use accentuation easily in this situation?
Thanks

**Comment by macmarrum** [comment]
Hi @jeapf 

Is your issue only in Freeplane? Can you enter Brazilian letters (accentuation) in other application on Kubuntu?

Maybe this will help: https://linuxnightly.com/how-to-change-keyboard-layout-ubuntu/

**Reply by jeapf** [reply]
Hi @macmarrum . Thank you for trying to help me.
Yes, I can use accentuation at other software.
For example, I can easily write “água”(water) in a Libre Office Writer file but if I try to do it in a Freeplane map I get only “gua”. It simply ignores the accentuation.
I am using a Brazilian Dell Inspiron 15 with Kubuntu 22.04 . The configured keyboard model is GENERIC 104 – PC KEY and the layout is “br Portuguese Brazil”

**Reply by macmarrum** [reply]
I use ArchLinux. I've set my keyboard to `br` and was able to enter `água` using `´` + `a` in Freeplane.

I have Ubuntu 22.04 in a VirtualBox. After setting `Portuguese (Brazil)` keyboard in Settings, it also works there.

Maybe there is a hotkey defined in your Freeplane that prevents accentuation. Please try to reset your profile, i.e. 
- in Freeplane menu, go to `Tools->Open user directory`
- in the file manager, go one level up
- close Freeplane
- rename `1.10.x` to `1.10.x-original`

Start Freeplane and try entering accentuations.

**Reply by jeapf** [reply]
Hi there. 
@macmarrum , thank you again for your time. I tried what you said but it didn’t work. 
It recreated the folder and kept the same behavior. I tried to install Freeplane 1.10.5 via snap in another Dell Notebook Inspiron N4030 I have which also runs Kubuntu 22.04 and the result was the same. Both notes Kubuntus are updated.
For knowledge for other readers: In BR PT we use accents(“´”, “`”, “^”, “~” ) above vowels only and we use “ç” in some cases. So, for example, “´”+ “a” = “á” and so on.
That said I did some more tests and found out that :
1 – “´” + vowel is completely ignored
2 – “ç” and “Ç” works perfectly
3 – “`”/ “^” / “~” + vowel  results in unaccented vowel
4 – The same result happens to capital letters
5 – I tried to change language inside Freeplane from “automatic” to English and PT-BR with the same result in both notes
6 – I use accentuation in other software in both notes without problems and I haven’t changed any Kubuntu’s default installed configuration about this

**Reply by jeapf** [reply]
It seems that what we call "crase" accent in PT-BR or inverted "´" causes some changes in the comments in the message before this one. Sorry for that.

**Comment by dpolivaev** [comment]
If I search https://www.google.com/search?q=accented+characters+java+kubuntu I get some links about it. Do any of them match your case?

**Reply by jeapf** [reply]
@dpolivaev thank you. I am going to try these solutions and post the results here
